appveyor.yml

```diff
@@ -19,14 +19,11 @@ environment:
 
 
 install:
-  # From https://github.com/nodejs/nan/blob/master/appveyor.yml
-  - ps: Install-Product node $env:nodejs_version
-  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:platform
```

Comment:
https://www.appveyor.com/docs/lang/nodejs-iojs/#testing-under-multiple-versions-of-nodejs-or-iojs